src/main/java/com/checkmarx/flow/config/JiraProperties.java

```diff
@@ -47,6 +49,7 @@
     private Integer maxJqlResults = 50;
     private List<String> statusCategoryOpenName = Arrays.asList("To Do", "In Progress");
     private List<String> statusCategoryClosedName = Arrays.asList("Done");
+    private String projectKeyScript;
```

Comment:
Probably it would be safer to just define this instead of @ Data?
```suggestion
    @Getter @Setter
    private String projectKeyScript;
```

Reply:
done